services/report/report_processor.py

```diff
@@ -159,13 +179,14 @@ def process_report(
     if report_type == "txt" and parsed_report[-11:] == b"has no code":
         # empty [dlst]
         return None
-    processors = get_possible_processors_list(report_type)
+    processors = get_possible_processors_list(report_type) if report_type else []
     for processor in processors:
         if processor.matches_content(parsed_report, first_line, name):
             sentry_metrics.incr(
                 "services.report.report_processor.parser",
                 tags={"type": type(processor).__name__},
             )
+            RAW_REPORT_SIZE.labels(processor=processor.name).observe(report.size)
```

Comment:
checked the callsite for this function to double-check that `report` here is indeed a single raw coverage report embedded inside of a larger Codecov upload https://github.com/codecov/worker/blob/a07ddef3a3ba329c9cda901d92fe345a7abb0375/services/report/raw_upload_processor.py#L159-L161